src/test/java/core/basesyntax/service/impl/FileReaderCvsImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderCvsImplTest {
+    private static final String TEMP_TEST_FILE_PATH =
+            "src/test/java/core/basesyntax/resources/test_activities.csv";
+    private final FileReader fileReader = new FileReaderCvsImpl();
+
+    @BeforeAll
+    public static void initAll() {
+        try {
+            Files.writeString(Paths.get(TEMP_TEST_FILE_PATH), "");
+        } catch (IOException e) {
+            throw new RuntimeException("Error cleaning data in file: " + TEMP_TEST_FILE_PATH);
+        }
+    }
+
+    private void writeDataInFile(String text) {
```

Comment:
Private ones (methods, classes) should be after public ones.

```suggestion
    private void writeDataToFile(String text) {
```